ownerdelete.user.js

```diff
@@ -163,6 +163,16 @@ dry.once("load", () => {
     }
     return file;
   };
+  async function getInfo(id) {
```

Comment:
I REALLY don't like that you use "let obj" i.e. a variable at all.
Just return the result or return null in the catch